src/pkgcheck/checks/visibility.py

```diff
@@ -231,6 +231,25 @@ def desc(self):
         return f"package uses old name which is source of pkgmove, rename into {self.new_name!r}"
 
 
+class OldPackageNameDep(results.VersionResult, results.Error):
+    """Package depends on old name which is source of pkgmove.
+
+    Package is depending ``${CATEGORY}/${PN}`` which is the source of a
+    pkgmove. It should be updated to the destination (new name) from
+    this repository or one of its master repositories.
+    """
+
+    def __init__(self, attr: str, dep: str, new_name: str, **kwargs):
+        super().__init__(**kwargs)
+        self.attr = attr
+        self.dep = dep
+        self.new_name = new_name
+
+    @property
+    def desc(self):
+        return f"{self.attr}: {self.dep!r} uses old package name which is source of pkgmove, rename into {self.new_name!r}"
```

Comment:
Bit wordier so I'm OK with the original even if it is a bit jarring:
```suggestion
        return f"{self.attr}: {self.dep!r} uses old package name which is the source of a pkgmove, rename into {self.new_name!r}"
```